scipy/special/tests/test_basic.py

```diff
@@ -1666,8 +1666,9 @@ def test_euler(self):
         eu0 = special.euler(0)
         eu1 = special.euler(1)
         eu2 = special.euler(2)   # just checking segfaults
-        assert_almost_equal(eu0[0],1,8)
-        assert_almost_equal(eu2[2],-1,8)
+        assert_allclose(eu0, [1], rtol=1e-15)
+        assert_allclose(eu1, [1, 0], rtol=1e-15)
```

Comment:
comparing against an expected value of ``0`` with ``atol=0`` usually gives problems. The original test had ``decimal=8`` - not sure if that was the right level, but you're tightening it a lot here. In any case should have ``atol`` not equal to zero.